_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,473 +856,2421 @@ If you need to brush up on how the IaC Library works, read the
 link:https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/[How to use
 the Gruntwork Infrastructure as Code Library] guide.
 
-=== Manual steps
-Start by completing the manual configurations describe above in <<manual_steps>>.
+=== Create the root account
+
+The first step is to create your root account. This account will be the parent of all of your other AWS accounts and
+the central place where you manage billing. You create this initial account manually, via a web browser:
+
+. Go to https://aws.amazon.com.
+. Click Create an AWS Account.
```

Comment:
This button doesn't appear for me. It appears only after "Sign In to the Console"

Reply:
@marinalimeira This is what I see in the top-right. Are we okay to resolve this one?

<img width="1515" alt="Screen Shot 2021-06-09 at 10 50 03 am" src="https://user-images.githubusercontent.com/178939/121285185-843a3900-c910-11eb-8d14-135e5fa62aba.png">